packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -247,7 +249,7 @@ def get_task_url(dcos_api_session, task_name, stream=False):
                                                                                          container_id)
 
 
-def validate_journald_cursor(c: str, cursor_regexp=None):
+def validate_journald_cursor(c: Any, cursor_regexp: Any = None) -> None:
```

Comment:
This case looks strange. Maybe we're expecting `bytes` here, since the regex is bytes.

Reply:
changed